pydicom/tests/test_util.py

```diff
@@ -117,7 +117,7 @@ def test_code_sequence(self):
         # ControlPointSequence
         elem = DataElement(0x300A0111, 'SQ', [])
         elem.value.append(Dataset())
-        elem[0].PatientID = '1234'
+        elem.value[0].PatientID = '1234'
         out = "\n"
         out += "# Control Point Sequence\n"
         out += "cp_sequence = Sequence()\n"
```

Comment:
Nitpick: why we are here, do you mind switching this multiple '+=' to an implied string concat in brackets? (or triple-quote perhaps). It would be easier to read, and a more pythonic form.

Reply:
Will do